yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -194,7 +165,7 @@ export class L2Block {
     const buf = serializeToBuffer(
       this.header.globalVariables,
       // TODO(#3868)
```

Comment:
#3868 have been closed. I'm guessing it just looks like this still as we never got back to it? I don't think we are really using this atm, but it should essentially be the `header` and the `archive` only, and not needing other values.

Reply:
Created [new issue](https://github.com/AztecProtocol/aztec-packages/issues/4844) for tackling this and linked that there instead of the stale TODO.